src/Components/CheatSheet.js

```diff
@@ -14,20 +21,57 @@ class CheatSheet extends Component {
         return resp.json();
       })
       .then(data => {
-        // console.log(data.kana);
-        this.handleShowData(data.kana);
+        this.setState({
+          kanaTable: data.kana
+        });
       });
   };
 
-  handleShowData = data => {
-    console.log(data);
+  handleGetHiragana = () => {
+    this.setState({
+      isHiraganaShown: true,
+      isKatakanaShown: false
+    });
+  };
+  handleGetKatakana = () => {
+    this.setState({
+      isHiraganaShown: false,
+      isKatakanaShown: true
+    });
   };
 
   render() {
+    const { isHiraganaShown, isKatakanaShown, kanaTable } = this.state;
     return (
       <section className="cheat-sheet" onClick={this.handleFetchData}>
         <UserNavBar />
-        <main className="cheat-sheet-container"></main>
+        <main className="cheat-sheet-container">
+          <div className="cheat-sheet-btns">
+            <button
+              className="cheat-sheet-hiragana-btn"
+              onClick={this.handleGetHiragana}
+            >
+              Hiragana
+            </button>
+            <button
+              className="cheat-sheet-katakana-btn"
+              onClick={this.handleGetKatakana}
+            >
+              Katakana
+            </button>
+          </div>
+          {isHiraganaShown &&
+            kanaTable.map(kana => (
+              <SingleSign kanaTable={kana} syllabary="hiragana" key={kana.id} />
```

Comment:
Instead of passing the whole kana object to SingleSign pass only the data you need: in this case kana.hiragana and kana.meaning - this way you do not need to put an extra if statement inside the SingleSign component